konf/site.yaml

```diff
@@ -15,12 +15,12 @@ env:
 
   - name: DISCOURSE_API_KEY
     secretKeyRef:
-      key: ubuntu-api-key
+      key: api-key
```

Comment:
Why the name change?

Reply:
I was having local issues using this key. But I can revert back once we officially review this PR

Reply:
Hmm. I didn't really mean you have  to revert it. I was curious. The reason I can imagine there'd be a property called `ubuntu-api-key` inside a secret called `discourse-api` would be because that secret contains many different discourse API keys. As in, it might have another key called `maas-api-key` or `canonical-api-key`.

If this isn't the case and it *only* contains the ubuntu discourse API key, then I would think the secret should be called `ubuntu-discourse-api` and then the property should be called `api-key`. Just wanted to see if these things were descriptively named, if that makes sense.